skyportal/handlers/api/instrument.py

```diff
@@ -428,131 +404,51 @@ async def get(self, instrument_id=None):
-                                InstrumentFieldTile.instrument_id == instrument.id,
-                                InstrumentFieldTile.instrument_field_id
-                                == InstrumentField.id,
-                                InstrumentFieldTile.healpix.overlaps(
-                                    LocalizationTile.healpix
-                                ),
-                            )
-                            .options(undefer(undefer_column))
-                        )
-                        .unique()
-                        .all()
-                    )
-                else:
-                    tiles = (
-                        (
-                            DBSession().execute(
-                                sa.select(InstrumentField).filter(
-                                    LocalizationTile.localization_id == localization.id,
-                                    LocalizationTile.probdensity >= min_probdensity,
-                                    InstrumentFieldTile.instrument_id == instrument.id,
-                                    InstrumentFieldTile.instrument_field_id
-                                    == InstrumentField.id,
-                                    InstrumentFieldTile.healpix.overlaps(
-                                        LocalizationTile.healpix
-                                    ),
+                    else:
+                        tiles = (
+                            (
+                                session.execute(
+                                    sa.select(InstrumentField).filter(
+                                        LocalizationTile.localization_id
+                                        == localization.id,
+                                        LocalizationTile.probdensity >= min_probdensity,
+                                        InstrumentFieldTile.instrument_id
+                                        == instrument.id,
+                                        InstrumentFieldTile.instrument_field_id
+                                        == InstrumentField.id,
+                                        InstrumentFieldTile.healpix.overlaps(
+                                            LocalizationTile.healpix
+                                        ),
+                                    )
                                 )
                             )
+                            .unique()
+                            .all()
                         )
-                        .unique()
-                        .all()
-                    )
-                data[ii]['fields'] = [
-                    {**tile.to_dict(), 'airmass': tile.airmass(time=airmass_time)}
-                    for tile, in tiles
-                ]
+                    data['fields'] = [
+                        {**tile.to_dict(), 'airmass': tile.airmass(time=airmass_time)}
+                        for tile, in tiles
```

Comment:
can remove comma if you change `execute` to `scalars`

Reply:
the sqlalchemy master.